rasa/train.py

```diff
@@ -399,15 +458,40 @@ async def _train_core_with_validated_data(
         domain, config = await asyncio.gather(
             file_importer.get_domain(), file_importer.get_config()
         )
-        async with telemetry.track_model_training(file_importer, model_type="core"):
-            await rasa.core.train(
-                domain_file=domain,
-                training_resource=file_importer,
-                output_path=os.path.join(_train_path, DEFAULT_CORE_SUBDIRECTORY_NAME),
-                policy_config=config,
-                additional_arguments=additional_arguments,
-                interpreter=interpreter,
-            )
+
+        if number_of_chunks > 1:
+            async with telemetry.track_model_training(
+                file_importer, model_type="core-in-chunks"
```

Comment:
Mhm, what about an additional boolean flag? That seems easier to filter for when analyzing the telemetry data 🤔

Reply:
Can I simply add it to the `TRAINING_STARTED_EVENT` or do I need to modify the event specification somewhere?

Reply:
You also need to document it in https://github.com/RasaHQ/rasa/blob/d095464e4daa182204c6af2a88a5165cfbbd95bf/docs/docs/telemetry/events.json#L11